react/components/ProductAvailability.tsx

```diff
@@ -3,19 +3,24 @@ import type { ProductTypes } from 'vtex.product-context'
 
 import LowStock from './LowStock'
 import HighStock from './HighStock'
+import ShowAvailable from './ShowAvailable'
 import Container from './Container'
 
 interface Props {
   threshold: number
   lowStockMessage?: string
   highStockMessage?: string
+  showAvailability?: 'stock'
```

Comment:
Make the default option a part of the enum too, like: `'disabled' | 'stock'`. This makes it easier to create form schemas on the site-editor and is cohesive with the rest of the apps.

```suggestion
  showAvailability?: 'disabled' | 'stock'
```